### PR TITLE
fix: border issue in charts and results table section

### PR DESCRIPTION
**Pull request by muhammadjufry**
Closes: #4558

### Description:
Fix border issue on table heading and table footer
Here's the video demo:
https://www.awesomescreenshot.com/video/16049471?key=f3a2c3d5eb3768ef4b75a1ed977d289c

Sorry for the video resolution

### CODE REVIEW

**Comment by rephus** [comment]
Hi Muhammad, it is nice to see you're contributing again, thanks !.

In your video you are not using frozen columns, it is good to see that it works with normal columns, but we also need to check it fixes the issue for frozen columns. Could you test it as well ? 

[Screencast from 31-03-23 08:42:06.webm](https://user-images.githubusercontent.com/1983672/229043365-2c77ccce-d8a2-4f50-ae08-706b9fca290e.webm)

**Comment by muhammadjufry** [comment]
Sure i will try to use frozen columns

**Comment by muhammadjufry** [comment]
Yes @rephus when i tried to use frozen columns the border doesn't look good, so i tried to fix that and it's fixed now!
Here's the video demo after fixing the border issue in frozen columns:
https://www.awesomescreenshot.com/video/16107515?key=8a4c0d4065fc91d0ee33bf343db8f46b

**Comment by rephus** [comment]
Nice, looks good, I think you're still missing one of the requirements from the ticket: 

- [ ]  frozen columns are divided by a shadow (it does not match our designs). We should elevate with darker BG colour and add a bit of a lighter shadow.

Right now I can't see which columns are frozen or not..

**Comment by muhammadjufry** [comment]
ok sure i will fix that now!

**Comment by muhammadjufry** [comment]
>  frozen columns are divided by a shadow (it does not match our designs). We should elevate with darker BG colour and add a bit of a lighter shadow.

Did you mean it like this:
Orders-Lightdash

**Comment by muhammadjufry** [comment]
@rephus Done. I just fixed it, here's the [video demo](https://www.awesomescreenshot.com/video/16290707?key=a634266d64c5b26f56140e414e826c34), as you can see on the video demo the frozen column has darker bg color and lighter border left and right
Please review it
Thank you

**Comment by IrakliJani** [comment]
hi @muhammadjufry, thanks for contributing, we'll review it early next week :raise

**Comment by muhammadjufry** [comment]
ok no problem, thanks

File: .env [hunk 1]
```diff
@@ -35,4 +35,4 @@ ALLOW_MULTIPLE_ORGS=false
 
 DBT_PROJECT_DIR=.
 
-SCHEDULER_ENABLED=true
+SCHEDULER_ENABLED=true
```
Comment (IrakliJani) [inline .env:38]: unnecesary change. can you revert?

File: packages/frontend/src/components/common/Table/Table.styles.ts [hunk 2]
```diff
@@ -71,29 +70,25 @@ export const Table = styled(HTMLTable)<{ $showFooter: boolean }>`
...
     }
-    .first-sticky-column {
-        box-shadow: lightgray -1px 0px 0px 0px, lightgray 0px 1px 0px 0px inset !important;
-    }
+
     .last-sticky-column {
-        border-right: 2px solid darkgray;
+        background-color: rgb(222, 222, 222);
+        border-right: 2px solid rgb(245, 245, 245);
+        border-left: 2px solid rgb(245, 245, 245);
```
Comment (IrakliJani) [inline packages/frontend/src/components/common/Table/Table.styles.ts:91]: we might need to update this with something much liter. can you use colours provided by the blueprint library?

**Review by IrakliJani — COMMENTED** [review]
minor comments

**Comment by muhammadjufry** [comment]
@IrakliJani Done i have reverted it the env code but i don't know why the github file changes still showing the old one, I have checked on my branch and it is same like the lightdash env code, and also i have changed the background and border color to blueprint color

Many thanks
Please review it

**Comment by muhammadjufry** [comment]
ok no problem, i will try again, many thanks

**Pull request closed without merging**